7.7.Поиск_ошибок_и_ревью_кода/7.7.3.py

```diff
@@ -0,0 +1,13 @@
+count = 0
+p = 1
+
+for i in range(1, 11):  
```

Comment:
1) Так как переменная цикла `i` не используется, то лучше назвать ее `_`.
2) Опять же, так как мы не используем значения последовательности `range()`, лучше передавать туда просто 10, будет столько же итераций.
```suggestion
for _ in range(10):  
```